services/harmony/test/ogc-api-coverages/get-coverage-rangeset.ts

```diff
@@ -343,6 +343,127 @@ describe('OGC API Coverages - getCoverageRangeset', function () {
     });
   });
 
+  describe('Using the "parameter_vars" pseudo-variable', function () {
```

Comment:
We should add a test for `"parameter_vars" specified, but no variables given`

Reply:
Good idea. Done.